### PR TITLE
Change ksceKernelGetFaultingProcess to official name

### PR DESCRIPTION
**Pull request by CreepNT**

### CODE REVIEW

File: include/psp2kern/kernel/threadmgr/debugger.h [hunk 1]
```diff
@@ -58,10 +58,10 @@ typedef struct SceThreadCpuRegisters
 /* Typedef for compatibility */
 typedef SceThreadCpuRegisters ThreadCpuRegisters;
 
-typedef struct SceKernelFaultingProcessInfo {
-	SceUID pid;
-	SceUID faultingThreadId; //!< Kernel UID of the faulting thread
-} SceKernelFaultingProcessInfo;
+typedef struct SceKernelThreadContextInfo {
+	ScePID processId;  //<! Process ID the thread scheduled on this CPU belongs to
+	SceUID threadId;   //<! Thread ID of the thread scheduled on this CPU
```
Comment (d3m3vilurr) [inline include/psp2kern/kernel/threadmgr/debugger.h:63]:
```suggestion
	ScePID process_id;  //<! Process ID the thread scheduled on this CPU belongs to
	SceUID thread_id;   //<! Thread ID of the thread scheduled on this CPU
```
can you change it to `snake_case`?

File: include/psp2kern/kernel/threadmgr/debugger.h [hunk 2]
```diff
@@ -87,13 +87,16 @@ int ksceKernelGetThreadIdList(SceUID pid, SceUID *ids, int n, int *copy_count);
...
- * @param      info  Output info
+ * This function can be called from an exception handler to obtain information about
+ * the thread that was interrupted.
  *
- * @return     Zero on success
+ * @param[out]  pInfo   Context information of the thread
+ *
+ * @return      Zero on success, < 0 on error
  */
-int ksceKernelGetFaultingProcess(SceKernelFaultingProcessInfo *info);
```
Comment (d3m3vilurr) [inline include/psp2kern/kernel/threadmgr/debugger.h:96]: I think it should keep and mark as `deprecated` to support backward compatibility.
@Princess-of-Sleeping can you review this?

Comment (Princess-of-Sleeping) [reply include/psp2kern/kernel/threadmgr/debugger.h:96]: Yeah, Or make macro is good.

**Comment by Princess-of-Sleeping** [comment]
Looks good for me.

**Pull request merged**